.github/workflows/main.yml

```diff
@@ -34,3 +32,14 @@ jobs:
         run: |
           echo Add other actions to build,
           echo test, and deploy your project.
+
+      # Runs installation commands
+      - name: Install npm dependencies and testing frameworks
+        run: |
+          npm install --save-dev jest babel-jest @babel/core @babel/preset-env puppeteer jest-puppeteer
```

Comment:
I think this command is redundant if all of the packages are listed as dependencies in package.json

Reply:
Good point, I'll make that change.